lambdas/thumbnail/index.py

```diff
@@ -47,7 +53,127 @@
     'additionalProperties': False
 }
 
-SUPPORTED_BROWSER_FORMATS = {'JPEG', 'GIF', 'PNG'}
+
+def generate_factor_pairs(n: int) -> List[Tuple[int]]:
```

Comment:
Can we move these to a separate file?

Reply:
The helper functions? Sure.

Reply:
Actually they are fine here for now; it's still a short file